CHANGES.md

```diff
@@ -23,6 +23,16 @@ OpenSSL 3.0
 
 ### Changes between 1.1.1 and 3.0 [xx XXX xxxx]
 
+ * Deprecated ERR_get_error_line(), as it's considered incomplete and confusing.
+   Dropped the new functions ERR_get_error_func() and ERR_get_error_data() for
+   the same reason, and because they were new in 3.0.0.
```

Comment:
This section in the CHANGES.md file has the title "Changes between 1.1.1 and 3.0". I don't think we need to mention changes that we made and then subsequently reverted - so I would drop this sentence. It's not relevant to someone upgrading from 1.1.1

Reply:
Also, the NEWS.md entry lists ERR_get_error_line_data(), ERR_peek_error_line_data(), ERR_peek_last_error_line_data() and ERR_func_error_string() - but they're not mentioned here.

Reply:
> Also, the NEWS.md entry lists ERR_get_error_line_data(), ERR_peek_error_line_data(), ERR_peek_last_error_line_data() and ERR_func_error_string() - but they're not mentioned here.

There's *another* entry further down in CHANGES that does, and it's with reference to that entry that I also mentioned the dropped functions.  But yeah, this has me think that I should probably edit that older CHANGES entry rather than this.

Reply:
I've done just that (especially noticing @t8m's thumbs-up)